### DISCUSSION TITLE
How to join the Slack channel?

### DISCUSSION THREAD

**Discussion by vtangstastic** [skills/secure-code-game#91] [General]
I somehow cannot access the program's Slack channel. When I tried to log in or sign up, I received this:

> You can use any account with the domain:
> github.com
> wearehackerone.com
> microsoft.com

**Comment by jkcso** [comment]
Thanks a lot for flagging this. I have now updated all the repository to include the correct slack URL which is [gh.io/securitylabslack](https://gh.io/securitylabslack)

**Reply by vtangstastic** [reply]
Thank you so much
